src/app/resources/resources.component.ts

```diff
@@ -113,6 +115,11 @@ export class ResourcesComponent implements OnInit, AfterViewInit, OnDestroy {
       this.emptyData = !this.resources.data.length;
       this.resources.paginator = this.paginator;
       this.dialogsLoadingService.stop();
+
+      const urlSelectedTags = this.route.snapshot.paramMap.get('tags').split('`');
```

Comment:
This will cause an error when the `tags` parameter doesn't exist (a case that should happen regularly)

Reply:
It actually does not.

Reply:
When loading the resources page this line throws an error.

![image](https://user-images.githubusercontent.com/9203229/63402196-6cdf3400-c3a8-11e9-87c7-472a50e6257d.png)